vlasiator.cpp

```diff
@@ -735,14 +723,10 @@ int main(int argn,char* args[]) {
               it != P::diagnosticVariableList.end();
          it++) {
             if (*it == "FluxB") {
```

Comment:
Just nuke these two away completely, as noted above, and also from datareductionoperator.cpp/.h, then we don't need this extra message.